Dockerfile

```
@@ -3,7 +3,7 @@ FROM ubuntu:20.04
 MAINTAINER Bartek Nowotarski <bartek@stellar.org>
 
 ENV STELLAR_CORE_VERSION 15.2.0-646.9334fcb.focal~cap21and40
-ENV HORIZON_VERSION 0.0.0~cap21and40-131
+ENV HORIZON_VERSION 0.0.0-158
```

pointing to the newer package I created to test the Horizon package from this [PR](https://github.com/leighmcculloch/stellar--go/pull/18) atm. I can change to point to the package that's built after that PR is merged

---

Do we need this deb change now, or can we keep it as is in this PR, so that the only thing changing with this PR is setting up the quick deploy button?
```suggestion
ENV HORIZON_VERSION 0.0.0~cap21and40-131
```

---

we do need to update to the newer Horizon package to be able to use the `--randomize-network-passphrase` flag for horizon. (tbh not sure which commit on master allows for it, I can do some digging if needed)

---

Hmm. `--randomize-network-passphrase` is part of the `stellar/quickstart` image, not Horizon, so it shouldn't matter which version of Horizon is in use?

---

I think the change needs to be a little different. Change the network passphrase in the cfg to be a variable that is always replaced then set it in the config. That fits better with the existing patterns. Can you fix this on master?

---

https://github.com/leighmcculloch/stellar--docker-stellar-core-horizon/pull/3/commits/2c6be5b9239e6743b2a2c35ed278c1ec2d4f1824

I'll add to master too

---

Yup the change looks good 👍🏻. Open a PR against master for it and I'll approve.

---

Then we can merge this PR. 🎉